backend/host_service.py

```
@@ -501,22 +507,17 @@ async def check_storage_availability(self, host_info: HostInfo, target_path: str
             )
             
-            if not is_valid:
-                shortage = total_required - available_bytes
-                result.error_message = f"Insufficient storage space. Need {format_bytes(total_required)} ({format_bytes(required_bytes)} + {format_bytes(safety_margin_bytes)} safety margin), but only {format_bytes(available_bytes)} available. Shortage: {format_bytes(shortage)}"
-            elif available_bytes < required_bytes * 1.5:
-                # Warning if less than 50% extra space
-                result.warning_message = f"Low storage space warning. Only {format_bytes(available_bytes)} available for {format_bytes(required_bytes)} required"
-            
-            logger.info(f"Storage validation for {safe_path}: {'✅ PASS' if is_valid else '❌ FAIL'} - {format_bytes(available_bytes)} available, {format_bytes(total_required)} required")
+            logger.info(f"Storage validation for {safe_path}: {'✓' if is_valid else '✗'} "
+                       f"(need {format_bytes(total_required)}, have {format_bytes(available_bytes)})")
```

Flake8 (7.2.0)

[error] continuation line under-indented for visual indent

(E128)